appcenter-file-upload-client-node/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "appcenter-file-upload-client-node",
-  "version": "1.1.0",
+  "version": "1.1.1",
```

Comment:
is this some bug fix ?

Reply:
Dependency updates. We only change the patch version for that.

Reply:
Dependency updates only

Reply:
ah cool